fixtures/meta-model/0.1.0.ts

```diff
@@ -0,0 +1,72 @@
+export interface ISemanticsTyped {
```

Comment:
Backwards compatibility is not a planned feature ;)

But seriously, let's do these kind of things when they start hurting, not before. And the thing is that this file is checked in, which means:

1. There should be a process for versioning the meta model, etc.
2. You need to take care with against which version you're running the tests.

I suggest reverting this, and just referencing the generated file.